api/src/main/java/jakarta/resource/ResourceException.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 1997, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025 Contributors to Eclipse Foundation. All rights reserved.
```

Comment:
Looks good, but same comment here about the copyright as in the other PR. Need to preserve the original copyright year,
```suggestion
 * Copyright (c) 1997, 2025 Contributors to Eclipse Foundation. All rights reserved.
```

Reply:
As in
- https://github.com/jakartaee/connectors/pull/152#discussion_r2047139618

Thanks for checking.